cmd/main.go

```diff
@@ -143,14 +149,17 @@ func handleSysMountArgs(args []string) []string {
 				fields := strings.SplitN(opt, "=", 2)
 				if flagName, ok := optionToCmdFlag[fields[0]]; ok {
 					newArgs = append(newArgs, fmt.Sprintf("--%s=%s", flagName, fields[1]))
-				} else if isBool, ok := cmdFlagsLookup[fields[0]]; ok && !isBool {
```

Comment:
we don't want to support `--debug=true` or `--debug=1`, it's better return an error for this case